cost/azure/unattached_volumes/README.md

```diff
@@ -0,0 +1,58 @@
+# Azure Unattached Volumes
+
+## What it does
+
+This Policy Template scans all volumes in the given account and identifies any unattached volumes that have been unattached for at least the number of user-specified days. If any are found, an incident report will show the volumes, and related information and an email will be sent to the user-specified email address.
```

Comment:
Please add some detail that log analtics is used to determine how long a volume is unattached.

Reply:
Added details about log analytics